zsl/application/service_application.py

```diff
@@ -56,6 +135,19 @@ def get_version(self):
         else:
             return AtteqServiceFlask.VERSION + ":" + v
 
-if not is_initialized():
-    raise Exception("Can not instantiate ServiceApplication object, the service is not initialized.")
-service_application = AtteqServiceFlask("zsl.application")
+    run_web = Flask.run
+    """Alias for Flask.run"""
+
+    def run_worker(self, *args, **kwargs):
```

Comment:
For the future maybe we could kick it from here and put it to the worker running code. Also we could inject the dependencies via DI and not use self.  We could also do the same with the run_web method. This I am mentioning because there are no methods like run_task or queue_task.

But it depends... The point is to decouple the running from the main application object. Also this object is used only for web (flask related things). Therefore we could split it into several parts.: worker, web and main application object not related to flask/web. Some day in the future. What do you say?